website_docs/automatic_instrumentation.md

```diff
@@ -0,0 +1,71 @@
+---
+Title: "Automatic Instrumentation"
+Weight: 3
+---
+
+Automatic instrumentation with Java uses a Java agent JAR that can be attached to any Java 8+ application. It dynamically injects bytecode to capture telemetry from many popular libraries and frameworks. It is the preferred way to capture telemetry data at the "edges" of an app or service, such as inbound requests, outbound HTTP calls, database calls, and so on. To instrument application code in your app or service, use [Manual Instrumentation](manual_instrumentation.md)
+
+## Setup
+
+Download the [latest version](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/).
+
+The JAR file (`opentelemetry-javaagent.jar`) contains the agent and all automatic instrumentation packages.
+
+Place the JAR in your preferred directory and launch it with your app:
+
+```
+java -javaagent:path/to/opentelemetry-javaagent.jar \
+     -jar myapp.jar
+```
```

Comment:
I would love to see us centralize the knowledge about how to set JVM args for different application servers, etc, in the opentelemetry docs, since this is a common stumbling block for new users.

E.g. Splunk otel javaagent has https://github.com/signalfx/splunk-otel-java/pull/472 (cc @theletterf), Microsoft otel javaagent has https://docs.microsoft.com/en-us/azure/azure-monitor/app/java-standalone-arguments (that I hacked together based on https://github.com/glowroot/glowroot/wiki/Where-are-my-application-server's-JVM-args%3F).

Reply:
Interesting point. How do you feel about that being a follow-up issue? I could do it, though it might take a bit to test out several options and verify it all works.

Reply:
> How do you feel about that being a follow-up issue?

oh yes, definitely 👍

Reply:
Done: https://github.com/open-telemetry/opentelemetry-java-docs/issues/5